### PR TITLE
added Italian Language

### PR DESCRIPTION
**Pull request by JostenSyon**
added italian language

### CODE REVIEW

**Comment by impeeza** [comment]
Cool, It's great see the community growing,  @solosky changing the order of the language definitions on fw/application/src/i18n/language.h, could be problematic?  when somebody upgrade the firmware and the new languaje for the index # now is other language, could be confusing for the end user.

**Comment by solosky** [comment]
> Cool, It's great see the community growing, @solosky changing the order of the language definitions on fw/application/src/i18n/language.h, could be problematic? when somebody upgrade the firmware and the new languaje for the index # now is other language, could be confusing for the end user.

Good point. New language should be added in end of the language enum list, because the language setting persisted as enum index to choose display language, if new language inserted in middle of the language enum, it will change the index means of actual language loaded in the initialize stage for language display.

File: fw/application/src/i18n/language.h [hunk 1]
```diff
@@ -15,6 +15,7 @@ typedef enum {
     LANGUAGE_EN_US,
     LANGUAGE_ZH_TW,
     LANGUAGE_ES_ES,
+    LANGUAGE_IT_IT,
```
Comment (solosky) [inline fw/application/src/i18n/language.h:18]: Please put the new languge defination in the end of the language enum.

**Review by solosky — CHANGES_REQUESTED** [review]
see my comments inline

File: fw/application/src/i18n/language.h (hunk 1 above)
Comment (JostenSyon) [reply fw/application/src/i18n/language.h:18]: done, even if i think that sooner or later language should be ordered better, I find confusing Europen, Eastern and Western languages all mixed up, usually all Eu language like spanish, franch, german, italian, portughese, are all close together, I understand English like first also mandarin chinese but for all the others... but ok, is just a personal taste ;)

Comment (solosky) [reply fw/application/src/i18n/language.h:18]: No, I think you did not resolve this. 
You should put LANGUAGE_IT_IT after LANGUAGE_FR_FR

**Review by solosky — CHANGES_REQUESTED** [review]

File: fw/application/src/i18n/language.h (hunk 1 above)
Comment (solosky) [reply fw/application/src/i18n/language.h]: Sorry, my mistake, please put LANGUAGE_IT_IT after LANGUAGE_PT_PT and before LANGUAGE_COUNT

**Comment by solosky** [comment]
Looks good, thanks for your contribution.

**Review by solosky — APPROVED** [review]

**Pull request merged**